package.json

```diff
@@ -11,11 +11,11 @@
 		"core-js": "^3.6.5",
 		"glob-parent": "5.1.2",
 		"vite": "^2.3.8",
-		"vue": "^3.0.0",
+		"vue": "^3.2.2",
 		"vue-tippy": "^6.0.0-alpha.30"
 	},
 	"devDependencies": {
-		"@vue/compiler-sfc": "^3.0.0"
+		"@vue/compiler-sfc": "^3.2.2"
```

Comment:
Is this still nessesarry? It's not just included with the move to vue 3.2?

Reply:
No clue lol